### DISCUSSION TITLE
suggestion: extend heightmap loader options with terrain paint

### DISCUSSION THREAD

**Discussion by Ryder17z** [OpenRCT2/OpenRCT2#15785] [Feature requests]
![image](https://user-images.githubusercontent.com/2000703/120095139-8fc57d00-c124-11eb-90c7-7c1629a890cb.png)

**Comment by duncanspumpkin** [comment]
@Ryder17z please provide more information. I've asked you to do this in the past as well. There is not enough information to know what it is that you are asking for.

**Comment by Ryder17z** [comment]
I thought it should be obvious as it is highlighted.

When a heightmap is imported, it is by default grass surface with rock walls. It would be good to have the same options as the other 3 tabs has.
